Samples/Blinky/Blinky-samd10xmini.lpi

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Same comments here as for the other lpi file, or did I oversee something?

Reply:
Correct: you did not overlook anything ... ;-)